README.md

```diff
@@ -79,6 +79,12 @@ Simple test that checks whether a file or directory exists. This indicates it in
 ### get(path:  string):  Promise<string  |  Buffer>
 Gets a file as a string/Buffer.
 
+### getFileOrFolderCreator(path: string):  Promise\<string\>
+Gets the username of the user that created the file or folder.
+
+### getObjectCreator(objectId: number | string):  Promise\<string\>
```

Comment:
What is an object?

Reply:
An object is a File or a Folder. The OCS methods use the id (objectId) rather than the path.